build/BUILD.bazel

```diff
@@ -1,20 +1,12 @@
 # gazelle:exclude parse.y.go
 load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
-load(":build_defs.bzl", "genfile_check_test", "go_yacc")
+load(":build_defs.bzl", "go_yacc")
 
 go_yacc(
     src = "parse.y",
     out = "parse.y.baz.go",
 )
 
-# parse.y.go is checked in to satisfy the Go community
-# https://github.com/bazelbuild/buildtools/issues/14
-# this test ensures it doesn't get stale.
-genfile_check_test(
-    src = "parse.y.go",
-    gen = "parse.y.baz.go",
-)
-
 go_library(
     name = "go_default_library",
     srcs = [
```

Comment:
The problem seems to be that "parse.y.baz.go" does not exist as a file in this directory and there's also nothing that generates it, now that the genfile_check_test is gone.

Reply:
it's generated by `go_yacc`

Reply:
I see, there's a testsuite in the root depending on the deleted test target.  I've removed it, but that error's pretty bad :(